### PR TITLE
unit tests RO, worker

### PR DESCRIPTION
**Pull request by bitProfessor**
first

### CODE REVIEW

File: tests/test_dexbot/test_worker.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        from fixtures import bitshares
+        self.worker.remove_offline_worker(self.TEST_CONFIG,bot,bitshares)    
+        # assert [] in self.worker.get_own_orders
+    def test_remove_offline_worker_data(self):
+        self.worker.remove_offline_worker_data(bot)
+    def test_do_next_tick(self):
+        pass
+if __name__ == '__main__':
+    cur_dir=os.path.dirname(__file__)
+    # path='/Users/jacking/Documents/GitHub/env/DEXBot/tests/test_dexbot/'
```
Comment (joelvai) [inline tests/test_dexbot/test_worker.py]: Remove commented code if not used.

File: tests/test_dexbot/test_relative_orders.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    #     markets=['BTS:CNY'],
+    #     accounts=['BTS0207'],
+    #     on_market=a.test_tick,
+    #     # on_account=self.on_account,
+    #     # on_block=self.on_block,
+    #     # bitshares_instance=self.bitshares
+    # )
+    # notify.listen()
+    cur_dir=os.path.dirname(__file__)
+    # path='/Users/jacking/Documents/GitHub/env/DEXBot/tests/test_dexbot/'
```
Comment (joelvai) [inline tests/test_dexbot/test_relative_orders.py]: Remove unused code if not used.

**Comment by bitProfessor** [comment]
Ok
On 04/08/2019 19:45, joelvai wrote:

@joelvai commented on this pull request.

In tests/test_dexbot/test_worker.py:

> +    def test_remove_market(self):
+        self.worker.remove_market(bot)
+        logging.info(self.worker.markets)
+        market=self.worker.markets
+        assert market == self.worker.markets
+    def test_remove_offline_worker(self):
+        from fixtures import bitshares
+        self.worker.remove_offline_worker(self.TEST_CONFIG,bot,bitshares)    
+        # assert [] in self.worker.get_own_orders
+    def test_remove_offline_worker_data(self):
+        self.worker.remove_offline_worker_data(bot)
+    def test_do_next_tick(self):
+        pass
+if __name__ == '__main__':
+    cur_dir=os.path.dirname(__file__)
+    # path='/Users/jacking/Documents/GitHub/env/DEXBot/tests/test_dexbot/'


Remove commented code if not used.

File: tests/test_dexbot/test_base.py [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    #     self.strategy_base.sort_orders_by_price()
+    # def test_update_gui_slider(self):
+    #     self.strategy_base.update_gui_slider()
+    # def test_update_gui_profit(self):
+    #     self.strategy_base.update_gui_profit()
+    # def tearDown(self):
+    #     pass
+#-------------------------test------------------
+if __name__=='__main__':
+    path='/Users/jacking/Documents/GitHub/env/DEXBot/tests/test_dexbot/test_strategies/'
```
Comment (joelvai) [inline tests/test_dexbot/test_base.py]: Hard coded path?

**Comment by bitProfessor** [comment]
Test_base.py is just starting to work.

On 04/08/2019 19:46, joelvai wrote:

@joelvai commented on this pull request.

In tests/test_dexbot/test_base.py:

> +    #     self.strategy_base.get_order(123)
+    # def test_get_updated_limit_order(self):
+    #     self.strategy_base.get_updated_limit_order([123])
+    # def test_purge_all_local_worker_data(self):
+    #     self.strategy_base.purge_all_local_worker_data('worker 2')
+    # def test_sort_orders_by_price(self):
+    #     self.strategy_base.sort_orders_by_price()
+    # def test_update_gui_slider(self):
+    #     self.strategy_base.update_gui_slider()
+    # def test_update_gui_profit(self):
+    #     self.strategy_base.update_gui_profit()
+    # def tearDown(self):
+    #     pass
+#-------------------------test------------------
+if __name__=='__main__':
+    path='/Users/jacking/Documents/GitHub/env/DEXBot/tests/test_dexbot/test_strategies/'


Hard coded path?

**Comment by bitProfessor** [comment]
Ok


On 04/08/2019 19:53, joelvai wrote:

@joelvai requested changes on this pull request.

To start off with, remove cache, .DS_Store and report.html files. Also make sure that all the Python files pass PEP8. Also the amount of commented code confuses me, why leave it there if not using it? I will re-review after these changes.

**Comment by thehapax** [comment]
@bitProfessor A few things to help the maintainer handle this easier. 

1. Just PR one unit test at the time. For example if this is for RO, the make a PR for RO only.
2. please squash commits.

We suggest you cancel this pull request and re-submit PR for clarity . Thanks much for your efforts.

**Comment by bitProfessor** [comment]
Ok


On 04/11/2019 04:51, octomatic wrote:

@bitProfessor A few things to help the maintainer handle this easier.

Just PR one unit test at the time. For example if this is for RO, the make a PR for RO only.
please squash commits.

We suggest you cancel this pull request and re-submit PR for clarity . Thanks much for your efforts.

**Comment by bitProfessor** [comment]
test

**Comment by bitProfessor** [comment]
test

**Pull request closed without merging**